File: week_11/day_2/objects/bear.js
var myBear = {
  name: "Rupert",
  type: "Polar Bear",
  belly: [],
  eat: function(food){
    this.belly.push(food);
  }
}

var food = "I am a piece of food."
myBear.eat(food);

console.log(myBear.belly);

File: week_6/day_1/starter_code/Bear.java
import java.util.ArrayList;

public class Bear{
  private String name;
  private ArrayList<Salmon> belly;

  public Bear(String name){
    this.name = name;
    this.belly = new ArrayList<Salmon>();
  }


  public String getName(){
    return this.name;
  }

  public int foodCount(){
    return belly.size();
  }

  public void eat(Salmon salmon){
    belly.add(salmon);
  }

  public void inTheWoods(){
    belly.clear();
  }

  public void remove(Salmon salmon){
    // for (Salmon fish : this.belly){
    //   if (fish.getName() == salmon.getName()){
        // System.out.println("The name of the fish is " + fish.getName());
        belly.remove(salmon);
    //   }
    // }
  }
}
